### PR TITLE
Fix EULA/licensing/signing issues post-merge

### PR DESCRIPTION
**Pull request by jaimergp**
# Description

## Type of change

- [X] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

* Installers: Update URLs to EULA 
* Docs: Fix markdown include in `community/licensing.md`
* Secrets: steps that need a secret are only run on scheduled runs. This means:
  * Conda packaging: Only upload to anaconda.org if the run is scheduled
  * Signing/notarization: only if scheduled
* Misc: Do not update `latest` from `conda_bundle` too (already in the old bundle CI -- we need to decide which one to keep).

More details:

After merging #4185 and #3555, we can confirm how the [EULA page looks like](https://napari.org/community/licensing.html). Unfortunately, the markdown include directive was a bit too literal.

image

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [X] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

**Review by Carreau — APPROVED** [review]

**Review by goanpeca — APPROVED** [review]

File: .github/workflows/make_bundle_conda.yml [hunk 1]
```diff
@@ -277,11 +279,18 @@ jobs:
...
-        if: runner.os == 'Windows'
+        if: runner.os == 'Windows' && github.event_name == 'schedule'
+        # We are signing with Apple's certificate to provide _something_
+        # This is not trusted by Windows so the warnings are still there, but curious users
+        # will be able to check it's actually us if necessary
         run: |
           Set-Content -Path "${{ runner.temp }}/certificate.b64.txt" -Value '${{ secrets.APPLE_APPLICATION_CERTIFICATE_BASE64 }}'
           certutil -decode "${{ runner.temp }}/certificate.b64.txt" "${{ runner.temp }}/certificate.pfx"
 
+          echo "CONSTRUCTOR_SIGNING_CERTIFICATE=${{ runner.temp }}/certificate.pfx" >> $Env:GITHUB_ENV
```
Comment (liu-ziyang) [inline .github/workflows/make_bundle_conda.yml:290]: the env configuration does not work for windows?

**Review by liu-ziyang — APPROVED** [review]

File: .github/workflows/make_bundle_conda.yml [hunk 2]
```diff
@@ -371,11 +373,12 @@ jobs:
...
 
-      - name: Update latest tag
-        uses: EndBug/latest-tag@latest
-        if: ${{ github.event_name == 'schedule' }}
-        with:
-          description: latest code released from nightly build
-          tag-name: latest
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # Choose which one to keep; the one in `make_bundle` or this one.
```
Comment (liu-ziyang) [inline .github/workflows/make_bundle_conda.yml:376]: we can drop this section completely, up to you

**Comment by liu-ziyang** [comment]
@napari/core-devs going to merge this soon to unblock the conda packaging effort. Since this merge breached having a 24h approval window according to our process. I would like to explicitly call it out. The scope for this PR is limited to conda packaging, and should not affect other napari functionality, and if there are comments we need to address, we will follow up in another PR very shortly

**Review by tlambert03 — APPROVED** [review]
LGTM!

File: .github/workflows/make_bundle_conda.yml (hunk 1 above)
Comment (jaimergp) [reply .github/workflows/make_bundle_conda.yml:290]: It does, but the presence of the keys changes behavior in the script, so we should only set these variables if scheduled, hence the change.

**Pull request merged**